how-to-use-azureml/machine-learning-pipelines/intro-to-pipelines/aml-pipelines-schedule-with-pipeline-parameter-built-in-functions.ipynb

```diff
@@ -0,0 +1,370 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Submit the Pipeline every 2 weeks on Monday and Wednesday at 6:30pm UTC\n",
+    "update_recurrence = ScheduleRecurrence(frequency=\"Week\", interval=2, week_days=[\"Monday\", \"Wednesday\"], time_of_day=\"18:30\")\n",
+    "update_pipeline_parameters = {\n",
+    "    \"start_time\": \"@addMinutes(trigger().scheduledTime, 10)\",\n",
+    "}\n",
+    "\n",
+    "pipeline_schedule.update(name=\"MyUpdatedPipelineSchedule\",\n",
+    "                description=\"Based on time, update pipeline with parameter expression\",\n",
+    "                pipeline_parameters=update_pipeline_parameters,\n",
+    "                recurrence=update_recurrence)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- Disable the schedule "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pipeline_schedule.disable()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Schedule a published pipeline endpoint with pipeline parameter expressions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define pipeline parameters\n",
+    "pipeline_parameters = {\n",
+    "    \"dayOfMonth\": \"@dayOfMonth('2022-01-01 08:00:00')\",\n",
+    "    \"dayOfWeek\": \"@dayOfWeek('2022-01-01 08:00:00')\",\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- Create a schedule with pipeline parameter expressions for published pipeline endpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pipeline_endpoint_schedule = Schedule.create_for_pipeline_endpoint(ws,\n",
+    "                                    name=\"MyPipelineEndpointSchedule\",\n",
+    "                                     pipeline_endpoint_id=pipeline_endpoint_id,\n",
+    "                                     experiment_name=\"Schedule-endpoint-pipeline-parameter-sample\",\n",
+    "                                     description=\"Create for pipeline endpoint with pipeline parameters\",      \n",
+    "                                     pipeline_parameters=pipeline_parameters,\n",
+    "                                     recurrence=recurrence\n",
+    "                                     ) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- Update a schedule with pipeline parameter expressions for published pipeline  \n",
+    "    **Note**: if users want to update pipeline without parameters, please republish the new pipeline to the pipeline endpoint, otherwise users need to update schedule to update pipeline with parameters."
```

Comment:
If you want to update pipeline parameter expression of published pipeline endpoint, you need to update the previous schedule in below way, as simply republishing the pipeline will not work.